src/_guides/language/analysis-options.md

```diff
@@ -121,19 +132,19 @@ analyzer:
 
 ## Enabling linter rules
 
-The analyzer package also provides a code linter.
-A [wide variety](http://dart-lang.github.io/linter/lints/)
-of linter rules are available. Linters tend to be
-non-denominational&mdash;rules don't have to agree with each other.
+The analyzer package also provides a code linter. A wide variety of
+[linter rules](http://dart-lang.github.io/linter/lints/)
+are available. Linters tend to be
+non denominational&mdash;rules don't have to agree with each other.
```

Comment:
It should be just "nondenominational", one word. (Actually, "non-denominational" is used often enough that it's justifiable, e.g. see https://en.wikipedia.org/wiki/Non-denominational, which uses both forms. But  "non denominational" would just be wrong.)

Most "non" words are hyphen free, as you can see from the big list at  http://www.merriam-webster.com/dictionary/non-.